src/gui/transferlistmodel.h

```diff
@@ -102,6 +106,14 @@ private slots:
 private:
     QList<BitTorrent::TorrentHandle *> m_torrentList;  // maps row number to torrent handle
     QHash<BitTorrent::TorrentHandle *, int> m_torrentMap;  // maps torrent handle to row number
+
+    // row text colors
+    QColor m_unknown, m_forcedDownloading, m_downloading, m_downloadingMetadata
```

Comment:
It's better to have some struct here (e.g. TorrentStateColors m_torrentStateColors). Or even `QHash<TorrentState, QColor>` since you want to access each color separately.

Reply:
I used `std::unordered_set`, QHash complains about missing qHash function.